nengo/dists.py

```diff
@@ -99,9 +102,9 @@ class Uniform(Distribution):
         If true, sample from a uniform distribution of integers. In this case,
         low and high should be integers.
     """
-    low = NumberParam()
-    high = NumberParam()
-    integer = BoolParam()
+    low = NumberParam('low')
```

Comment:
+1 to @jgosmann. It's not ideal, but it's not near as bad as the `label` for NengoObjects situation, since that concerned every model ever, and this only concerns advanced users and devs. If someone can come up with a better solution in the future, by all means, but not worth holding up this PR for.